### PR TITLE
Jl/offer currency and fallback

### PR DESCRIPTION
**Pull request by jrlarano**
Relates to: https://github.com/tjek/octopus/issues/556

### CODE REVIEW

File: lib/kits/core-ui/components/common/offer-list.ts [hunk 1]
```diff
@@ -57,6 +59,7 @@ const OfferList = ({
         id?: string;
         businessId?: string;
     };
+    sgnData: any;
```
Comment (klarstrup) [inline lib/kits/core-ui/components/common/offer-list.ts]: Please put the actual type, this one is already well-known: https://github.com/tjek/tjek-js-sdk/blob/develop/lib/kits/core-ui/paged-publication.ts#L41

File: lib/kits/core-ui/components/common/shopping-list.ts [hunk 2]
```diff
@@ -113,13 +114,15 @@ const defaultTemplate = `\
     </div>\
 `;
 
-const ShoppingList = ({template}) => {
+const ShoppingList = ({template, scriptEls, sgnData}) => {
```
Comment (klarstrup) [inline lib/kits/core-ui/components/common/shopping-list.ts]: Please put the actual types, these are already well-known: 

https://github.com/tjek/tjek-js-sdk/blob/develop/lib/kits/core-ui/paged-publication.ts#L41 

https://github.com/tjek/tjek-js-sdk/blob/develop/lib/kits/core-ui/components/paged-publication/main-container.ts#L60

**Review by klarstrup — APPROVED** [review]
Fine but stop slacking on the types

**Comment by jrlarano** [comment]
@klarstrup sorry about that. I've updated it with types 😅

**Comment by klarstrup** [comment]
Live now on all channels https://github.com/tjek/tjek-js-sdk/releases/tag/release-18.09.2024-14.07

**Pull request merged**